examples/longhorn/zarf.yaml

```diff
@@ -6,6 +6,32 @@ components:
   - name: longhorn
     required: true
     description: "Deploy Longhorn into a Kubernetes cluster.  https://longhorn.io"
+    actions:
+      # Run the Longhorn Environment Check on this cluster's nodes.
+      onDeploy:
+        before:
+          - env: 
+            - "PATH=$PATH:./"
+          - cmd: ./environment_check.sh
+      # Set the delete confirmation flag for Longhorn
+      onRemove:
+        before:
+          - env:
+            - "PATH=$PATH:./"
+          - cmd: "kubectl -n longhorn-system patch -p '{\"value\": \"true\"}' --type=merge lhs deleting-confirmation-flag"
+    files:
+      # jq
+      - source: https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+        target: jq
+        executable: true
+      # kubectl
+      - source: https://dl.k8s.io/release/v1.26.0/bin/linux/amd64/kubectl
```

Comment:
this should probably be dropped now since when vendor kubectl in I think?

Reply:
Removed per previous comment